BioArchLinux/snpeff/lilac.yaml

```diff
@@ -0,0 +1,11 @@
+maintainers:
+  - github: kbipinkumar
+    email: bipin@ccmb.res.in
+build_prefix: extra-x86_64
+post_build_script:
+  git_add_files('PKGBUILD')
+  git_commit()
+  update_aur_repo()
+update_on:
+  - source: aur
+    aur: snpeff
```

Comment:
`update_aur_repo()` and `update_on:aur` conflict with each other (cyclic), maybe you want `update_on: github`?